lib/usdUfe/ufe/UsdUndoableSetSceneItemMetadataCommand.cpp

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "UsdUndoableSetSceneItemMetadataCommand.h"
+
+#include "Utils.h"
+#include "private/Utils.h"
```

Comment:
The first Utils.h should be included via <usdUfe/xxx>.
The private one is not needed.